### ISSUE TITLE
add nightly build

### ISSUE THREAD

**Issue by alisterd51** [melonDS-emu/melonDS#2161]
I think we should create a new job to generate a build based on the master branch with a `nightly` tag. This would allow everyone to benefit from the new features without waiting for a release.

I’m available to create a merge request if you're interested.

**Comment by nadiaholmquist** [comment]
There are already builds of every commit to master, look in the Actions tab or the build status next to the commit.

**Comment by Atem18** [comment]
@nadiaholmquist There is no nightly flatpak right ?

**Comment by nadiaholmquist** [comment]
No, not currently. I have thought of doing it, but I've not quite figured out how exactly to do it well, and the documentation was quite limited last I looked.

**Comment by Atem18** [comment]
Yeah I saw the appimage but I am using emudeck so I thought replacing the flatpak directly would do it.

You could also publish to flathub beta.
